### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by sharzie**
add some shit

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1 +1 @@
-# RR 
+# RR 1
```
Comment (shardool) [inline Readme.md]: good u added 1

File: Readme.md [hunk 2]
```diff
@@ -1 +1 @@
-# RR
```
Comment (shardool) [inline Readme.md]: but why just RR ?

File: Readme.md (hunk 1 above)
Comment (sharzie) [inline Readme.md]: yeah

File: Readme.md (hunk 2 above)
Comment (shardool) [inline Readme.md]: you could add something after RR

**Pull request merged**